tests/benchdnn/doc/driver_ip.md

```diff
@@ -21,6 +21,11 @@ where *ip-knobs* are:
             Refer to [tags](knobs_tag.md) for details.
  - `--dtag={any [default], ...}` -- physical dst memory layout.
             Refer to [tags](knobs_tag.md) for details.
+ - `--bia-dt={undef [default], f32, bf16, f16, ...}` -- bias data type.
+            To run Inner Product without bias, use `undef` data type (default).
+            `--dir=FWD_B` will set `--bia-dt` to `f32` to preserve compatibility
```

Comment:
Is there any particular reason why you don't mention `BWD_WB`?

Reply:
Memory loss in that particular moment. Thanks for the catch, updated.